OpenSim/Common/ComponentPath.h

```diff
@@ -51,21 +51,21 @@ class OSIMCOMMON_API ComponentPath : public Path {
     
     /// Construct a ComponentPath from a string. This will clean up the
     /// path, removing and resolving "." and ".." when possible.
-    ComponentPath(const std::string path);
+    ComponentPath(const std::string& path);
 
     /// Constructor a ComponentPath from a vector that contains all subtree
     /// node names and a bool that indicates if the path is an absolute path.
-    ComponentPath(std::vector<std::string> pathVec, bool isAbsolute);
+    ComponentPath(std::vector<std::string>& pathVec, bool isAbsolute);
```

Comment:
Sorry I neglected to say this in my previous review, but this should also be const (`const std::vector<std::string>&`).